aws-ecs/bastion.userdata.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+# updates
+yum update -y
+amazon-linux-extras install postgresql10
+yum install htop jq nc -y
```

Comment:
Why nc?

Reply:
I thought this was a machine for testing, so why not? Probing for open ports is convenient. Should we remove it?